components/feature/qr/src/test/java/mozilla/components/feature/qr/QrFragmentTest.kt

```diff
@@ -617,4 +619,68 @@ class QrFragmentTest {
         assertSame(bitmap, imageCaptor.allValues[0])
         assertSame(invertedBitmap, imageCaptor.allValues[1])
     }
+
+    @Test
+    @Suppress("DEPRECATION")
+    fun `getScreenRotation returns correct display rotation`() {
```

Comment:
nit: Let's try to describe the expected behavior that we are testing instead of the method that are getting executed, this will help to self document the expected functionality and if the name of the functions changes tests won't get outdated. We could use [GIVEN WHEN, THEN](https://martinfowler.com/bliki/GivenWhenThen.html).

Maybe:
````kOTLIN
  `GIVEN the device is on 90 degrees 
   WHEN getting the device rotation on a device below SDK 30 
   THEN the roation should be 90 degrees` 
````